deploy/helms/oauth2-proxy/values.yaml

```diff
@@ -5,8 +5,8 @@
 replicaCount: 1
 
 image:
-  repository: docker.io/sath89/oauth2_proxy
-  tag: kubernetes
+  repository: legionplatform/k8s-oauth2-proxy-docker
```

Comment:
do we really need "-docker" suffix for this image?

Reply:
Not needed. Removed suffix.